docs/background/make_models.rst

```diff
@@ -0,0 +1,53 @@
+.. _background_make_models:
+
+Make models
+===========
+
+Gammapy tools to produce background models.
+
+Make cube models
+----------------
+
+The ``gammapy-make_bg_cube_models`` command line tool can be used to produce
+background cube models from the data files at a given path location.
+
+For more details, please refer to `~gammapy.scripts.make_bg_cube_models`.
+
+Examples
+~~~~~~~~
+
+The ``gammapy-make_bg_cube_models`` tool has a few options. For a full list of options, please use:
+
+  .. code-block:: bash
```

Comment:
Remove space before the `..` This doesn't need to be indented:
![screen shot 2015-08-12 at 11 40 10](https://cloud.githubusercontent.com/assets/852409/9220988/eadcf5de-40e6-11e5-8817-4e84ebfc23aa.png)